docs/getting-started.md

```diff
@@ -1,21 +1,186 @@
+
+<img src="getting-started-imgs/help.png" width="75%">
+
+Let's try out a few of these shortcuts!
+
++ First, type <kbd>P</kbd> to view your Private Messages (PMs).
+
++ Second, type <kbd>#</kbd> to view all messages you were mentioned (@'ed) in.
+
++ Third, type <kbd>/</kbd> to jump to the search bar at the top of the screen in the middle column. Type <kbd>esc</kbd> to move your cursor out of the search box so we can use our keyboard shortcuts again (otherwise you'd just be typing them in the search box!).
+
+
+## Narrowing
+
+You'll be hearing the term 'narrow' a lot in Zulip. What does it mean?
+
++ **Noun**: A narrow is a set of filters for Zulip messages, that can be based on many different factors (like sender, stream, topic, search keywords, etc.).
++ **Verb**: The process of navigating to a different narrow. For example, to go from viewing a stream to a topic within that stream. There are several different ways to 'narrow to a narrow' so to speak. For example, you can use the keyboard shortcuts as we did in the section above. When you used the keyboard shortcut <kbd>P</kbd> to view your Private Messages (PMs), you 'narrowed' to the Private Messages narrow.
+
+### Your Current Narrow
+
+Your current narrow is displayed at the top of Zulip Terminal in the middle column, see the image below for details. If you ever get lost, just look here.
+
+<img src="getting-started-imgs/current-narrow.png" width="75%">
+
+### Narrow to a Stream
+To 'narrow to a stream' means to set the middle column to only show messages present in that stream. Let's try narrowing to the [# test here](https://chat.zulip.org/#narrow/stream/7-test-here) stream. [Move your Cursor](#Your-Cursor) to the left column of Zulip Terminal and then move your cursor down (<kbd>Down</kbd>/<kbd>j</kbd>) until it rests on "# test here". Press <kbd>Enter</kbd> to narrow to the stream.
+
+Check out the top of the middle column, your current narrow should now be "test here" like in the image below.
+
+*Note: The colors in your terminal may vary from the ones in the screenshot.*
+
+<img src="getting-started-imgs/narrow-stream.png" width="75%">
+
+### Narrow to a Topic
+
+First, let's move our cursor back to the [# test here](https://chat.zulip.org/#narrow/stream/7-test-here) stream in the left column - otherwise the following keyboard shortcut won't work. With our cursor on the [# test here](https://chat.zulip.org/#narrow/stream/7-test-here) stream, type <kbd>t</kbd> to toggle on a list of topics within that stream. Move your cursor to a random topic and then press <kbd>Enter</kbd> to narrow to that topic. You've successfully narrowed to a topic! Notice how at the top of the middle column your current narrow is displayed as "test here: topic you chose". 
```

Comment:
> "test here: topic you chose"

nit: Is it possible to use the specific right-arrow character here instead of a colon? 
![image](https://user-images.githubusercontent.com/22248748/110875935-90c4dc00-82a4-11eb-97f7-2e015bb21aa0.png)

Reply:
good idea!!